app/grants/tasks.py

```diff
@@ -108,7 +110,7 @@ def update_grant_metadata(self, grant_id, retry: bool = True) -> None:
 
 
 @app.shared_task(bind=True, max_retries=1)
-def process_grant_contribution(self, grant_id, grant_slug, profile_id, package, retry: bool = True) -> None:
+def process_grant_contribution(self, grant_ids_list, profile_id, post_request, retry: bool = True):
```

Comment:
rename this to `process_grant_contributions` and could you add a comment saying what this function does